WearTilesKotlin/app/build.gradle

```diff
@@ -64,6 +64,7 @@ android {
 
 dependencies {
     implementation libs.androidx.lifecycle.runtime.ktx
+    implementation libs.androidx.lifecycle.viewmodel.ktx
```

Comment:
Something is pulling in different versions of androidx.lifecycle jars which had some class moves.